lib/froala-editor-sdk/file.rb

```diff
@@ -12,19 +12,18 @@ class File
             allowedExts: [".txt", ".pdf", ".doc", ".json", ".html"],
             allowedMimeTypes: [ "text/plain", "application/msword", "application/x-pdf", "application/pdf", "application/json","text/html" ]
         },
-        resize: nil
+        resize: nil,
+        file_upload_path: 'public/uploads/files',
+        file_access_path: '/uploads/'
```

Comment:
I would put just this one here.

Reply:
@stefanneculai But moving this field to the options is what is required in order to fix this issue. Thanks.

Reply:
We shall move only `file_access_path` to options. `file_upload_path` would remain as it is now, param for the `upload` method. Thanks.